cairo-hints/scarb-hints-test/main.rs

```diff
@@ -110,8 +128,8 @@ fn check_scarb_version(metadata: &Metadata) {
         .to_string();
     if app_version != scarb_version {
         println!(
-            "warn: the version of cairo-test does not match the version of scarb.\
-         cairo-test: `{}`, scarb: `{}`",
+            "warn: the version of scarb-hints-test does not match the version of scarb. \
+         scarb-hints-test: `{}`, scarb: `{}`",
             app_version, scarb_version
```

Comment:
It's never going to match - we will not be releasing a new version of cairo-hints per each scarb version. We should simply remove this check.

Reply:
We should remove this check?